packages/rocketchat-ui/client/lib/notification.js

```diff
@@ -79,15 +79,27 @@ const KonchatNotification = {
 	newMessage(rid) {
 		if (!Session.equals(`user_${ Meteor.userId() }_status`, 'busy')) {
 			const user = Meteor.user();
-			const newMessageNotification = user && user.settings && user.settings.preferences && user.settings.preferences.newMessageNotification || 'chime';
-			const sub = ChatSubscription.findOne({ rid }, { fields: { audioNotification: 1 } });
-			if (sub && sub.audioNotification !== 'none') {
-				if (sub && sub.audioNotification) {
-					const [audio] = $(`audio#${ sub.audioNotification }`);
-					return audio && audio.play && audio.play();
-				} else if (newMessageNotification !== 'none') {
-					const [audio] = $(`audio#${ newMessageNotification }`);
-					return audio && audio.play && audio.play();
+
+			if (user && user.settings && user.settings.preferences) {
+				const newMessageNotification = user.settings.preferences.newMessageNotification || 'chime';
```

Comment:
If there is no `user.settings.preferences` then there will not have notifications?